kolibri/plugins/facility/assets/src/views/ManageClassPage/index.vue

```diff
@@ -35,70 +35,48 @@
         </KGridItem>
       </KGrid>
 
-      <CoreTable
-        :dataLoading="dataLoading"
-        :emptyMessage="$tr('noClassesExist')"
+      <KTable
+        :headers="tableHeaders"
+        :rows="tableRows"
+        :caption="$tr('tableCaption')"
+        :useLocalSorting="true"
       >
-        <caption class="visuallyhidden">
-          {{
-            $tr('tableCaption')
-          }}
-        </caption>
-        <template #headers>
-          <th>{{ coreString('classNameLabel') }}</th>
-          <th>{{ coreString('coachesLabel') }}</th>
-          <th>{{ coreString('learnersLabel') }}</th>
-          <th>
-            <span class="visuallyhidden">
-              {{ coreString('userActionsColumnHeader') }}
-            </span>
-          </th>
+        <template #header="{ header }">
+          {{ header.label }}
         </template>
-        <template #tbody>
-          <transition-group
-            tag="tbody"
-            name="list"
-          >
-            <tr
-              v-for="classroom in sortedClassrooms"
-              :key="classroom.id"
+        <template #cell="{ content, rowIndex, colIndex, row }">
```

Comment:
I'm getting the error: `Invalid prop: type check failed for prop "content". Expected String, Number, got Object`. It might be best to include `Object` in the content type or only a string should be passed.

Reply:
Added object in the `content` type.